discovery-provider/src/tasks/index.py

```diff
@@ -267,12 +274,12 @@ def revert_blocks(self, db, revert_blocks_list):
                 parent_hash = default_config_start_hash
 
             # Update newly current block row and outdated row (indicated by current block's parent hash)
-            session.query(Block).filter(Block.blockhash == parent_hash).update(
-                {"is_current": True}
-            )
             session.query(Block).filter(Block.blockhash == revert_hash).update(
                 {"is_current": False}
             )
+            session.query(Block).filter(Block.blockhash == parent_hash).update(
```

Comment:
is there a difference in switching the order in which you update the parent_hash or revert_hash?

Reply:
this was actually a change on master. i didn't make this change. i did a git pull origin master and this got included here. it was to re-enable revert blocks